pkg/registries/quay/quay_test.go

```diff
@@ -0,0 +1,135 @@
+package quay
+
+import (
+	"testing"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/features"
+	"github.com/stackrox/rox/pkg/testutils/envisolator"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestValidate(t *testing.T) {
+	envIsolator := envisolator.NewEnvIsolator(t)
+	envIsolator.Setenv(features.QuayRobotAccounts.EnvVar(), "true")
+
+	if !features.QuayRobotAccounts.Enabled() {
```

Comment:
You can use https://github.com/stackrox/stackrox/blob/master/pkg/testutils/feature_flag.go#L24 if you want